### PR TITLE
Statefulsets performance 

### PR DESCRIPTION
**Pull request by paigerube14**
This is adding a set automation script for the statefulsets performance [TC](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-15356)

### CODE REVIEW

**Comment by paigerube14** [comment]
I used 6 worker nodes at m5.4xlarge size

**Comment by mffiedler** [comment]
There's a stray empty file from pod_density in the commit.

**Comment by paigerube14** [comment]
> There's a stray empty file from pod_density in the commit.

I think I need the init file to be able to use one of the functions in the pod_density folder. https://github.com/openshift/svt/blob/f9cc08ac7371ed01cc4a5bdf5a0500c8ab065288/openshift_scalability/scripts/statefulset_perf.sh#L10

**Comment by mffiedler** [comment]
Got it - missed that.  Disregard comment.

File: openshift_scalability/scripts/statefulset_perf.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  COUNTER=0
+  nodes=$(oc get nodes -l 'node-role.kubernetes.io/worker='  | awk '{print $1}'| grep -v NAME )
+  for node_name in $nodes
+  do
+    oc describe node $node_name
+
+  done
+}
+
+CONFIG_FILE=../config/pyconfigStatefulSet.yaml
```
Comment (mffiedler) [inline openshift_scalability/scripts/statefulset_perf.sh:70]: Path math problem again.   This yaml file has ./content/statefulset-pv-template.json in it which results in the path "./content/statefulset-pv-template.json".    Easy fix is to update this yaml to be ../content

Comment (paigerube14) [reply openshift_scalability/scripts/statefulset_perf.sh:70]: should be fixed now

**Pull request merged**